lib/preprocess/raw.js

```diff
@@ -0,0 +1,27 @@
+const { nodeListToArray } = require('../utils')
+
+// FIXME: rename docxRaw to docxRun or docxRawRun?
+const regexp = /{{docxRaw [^{}]{0,500}}}/

```

Comment:
we will like the xml to be passed as a named paratemer of the helper, something like this

```
{{docxRaw xml=<xml content here>}}
```

Reply:
Done.